mobile/src/navigation/app/app.navigation.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from 'react';
+import { ParamListBase } from '@react-navigation/native';
+import {
+  createDrawerNavigator,
+  DrawerNavigationOptions,
+} from '@react-navigation/drawer';
+
+import { Overview } from '~/components/overview/overview';
+import { Courses } from '~/components/courses/courses';
+import { Mentors } from '~/components/mentors/mentors';
+import { MyEducation } from '~/components/my-education/my-education';
+import { Billing } from '~/components/billing/billing';
+import { Settings } from '~/components/setting/setting';
+import { CustomDrawerContent } from './components/drawer-content/drawer-content';
+import { AppScreenName, AppColor } from '~/common/enums/enums';
+
+const Drawer = createDrawerNavigator<ParamListBase>();
```

Comment:
Create AppNavigationParamList and define all screens there
```suggestion
const Drawer = createDrawerNavigator<AppNavigationParamList>();
```